definitions/variable/technology/electricity-operation.yaml

```diff
@@ -12,360 +12,71 @@
 #   - Flows in transmission lines
 # Demand not served
 
-- Active Power|Electricity|Biomass:
-    description: Active power of Biomass power plants
-    unit: MWh
-- Active Power|Electricity|Biomass|Traditional:
-    description: Active power of  Traditional Biomass power plants biomass from local
-      and pre-existing resources mostly waste :domestic and agricultural waste, forest
-      residues.
-    unit: MWh
-- Active Power|Electricity|Biomass|New and imported:
-    description: Active power of New+Imported biomass power plants biomass from dedicated
-      energy crops and biomass imports.
-    unit: MWh
-- Active Power|Electricity|Nuclear:
-    description: Active power of nuclear power plants
-    unit: MWh
-- Active Power|Electricity|Gas:
-    description: Active power of Gas power plants
-    unit: MWh
-- Active Power|Electricity|Gas|OCGT:
-    description: Active power of Open Cycle Gas Turbine power plants
-    unit: MWh
-- Active Power|Electricity|Gas|CCGT:
-    description: Active power of Combined Cycle Gas Turbine power plants
-    unit: MWh
-- Active Power|Electricity|Coal:
-    description: Active power of coal power plants
-    unit: MWh
-- Active Power|Electricity|Lignite:
-    description: Active power of Lignite power plants
-    unit: MWh
-- Active Power|Electricity|Oil:
-    description: Active power of oil power plants
-    unit: MWh
-- Active Power|Electricity|Geothermal:
-    description: Active power of Geothermal power plants
-    unit: MWh
-- Active Power|Electricity|Hydro|Reservoir:
-    description: Active power of "simple Hydro reservoir " units
-    unit: MWh
-- Active Power|Electricity|Hydro|Pumped Storage:
-    description: Active power of "Pumped Storage " units
-    unit: MWh
-- Active Power|Electricity|Hydro|Run of River:
-    description: Active power of "Run of River" units
-    unit: MWh
-- Active Power|Electricity|Hydro|Ocean:
-    description: Active power of Ocean power plants
-    unit: MWh
-- Active Power|Electricity|Solar:
-    description: Active power of Photovoltaic power plants
-    unit: MWh
-- Active Power|Electricity|Wind|OnShore:
-    description: Active power of  Wind Onshore power plants
-    unit: MWh
-- Active Power|Electricity|Wind|OffShore:
-    description: Active power of  Wind OffShore power plants
-    unit: MWh
+- Active Power|Electricity|<Fuel>:
+    description: Active power of <Fuel> power plants
+    unit:
+    - GWh
+
 - Active Power|Electricity|Load Curtailment:
     description: Energy curtailed - from the system point of view is equivalent to
       generation of Active Power
-    unit: MWh
+    unit:
+    - GWh
+
 - Active Power|Electricity|Load Shifting:
     description: Energy shifted - from the system point of view is equivalent to generation
       of Active Power
-    unit: MWh
+    unit:
+    - GWh
+
 - Active Power|Electricity|Electricity Storage:
```

Comment:
shoudn't this be Active Power|Electricity|Electricity Storage System|<type of storage system>
in the scenarios we already have 3 kinds of storage systems with redoxFlow, compressed air.....

Reply:
In the beginning, I tried to include the following technologies to `tag_fuel_types.yaml`, in order to be included when the tag <Fuel> is used: 

  - Energy Storage System:
      description: Energy Storage System, excl. hydro storage  
  - Energy Storage System|Lithium-Ion:
      description: Energy Storage System based on the lithium-ion batteries systems
  - Energy Storage System|Redox Flow:
      description: Energy Storage System based on the redox flow batteries systems
  - Energy Storage System|Compressed Air:
      description: Energy Storage System based on the compressed air energy storages systems

@sandrinecharousset, @danielhuppmann, Should we include something similar to `tag_fuel_types.yaml`?